platform/broadcom/platform-modules-delta.mk

```diff
@@ -40,4 +40,4 @@ DELTA_AGC032_PLATFORM_MODULE = platform-modules-agc032_$(DELTA_AGC032_PLATFORM_M
 $(DELTA_AGC032_PLATFORM_MODULE)_PLATFORM = x86_64-delta_agc032-r0
 $(eval $(call add_extra_package,$(DELTA_AG9032V1_PLATFORM_MODULE),$(DELTA_AGC032_PLATFORM_MODULE)))
 
-SONIC_STRETCH_DEBS += $(DELTA_AG9032V1_PLATFORM_MODULE)
+SONIC_BUSTER_DEBS += $(DELTA_AG9032V1_PLATFORM_MODULE)
```

Comment:
we can do that in a different pr.

Reply:
Done. @lguohan Shall i open a different PR for check in these changes ? or is there already one i can use ?